hack/make/test-unit

```diff
@@ -48,8 +48,8 @@ bundle_test_unit() {
 			| grep -v github.com/docker/docker/man \
 			| grep -v github.com/docker/docker/integration-cli)
 	fi
-
 	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS $pkg_list
+	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS github.com/docker/docker/pkg/loopback -test.root
```

Comment:
Test cases of all the packages are executed with the same flag. So if we provide `-test.root` to earlier command we will receive and error `flag provided but not defined: test.root` for all of the packages and we will have to define this flag in all packages. Which is why it was decided to be put explicitly for this package only.